src/static/js/main.js

```diff
@@ -46,17 +46,58 @@ $(function () {
       contentType: 'application/json'
     }).done(function (response) {
       $(form).find('button[type=submit]').toggleClass('loading')
-      // This is used allow each template to implement its own response handler
-      // if (FORM_HANDLERS[$(form).attr('name')]) {
-      //   FORM_HANDLERS[$(form).attr('name')](form, response)
-      // }
+     
 
-      console.log('asdasdasd', THE_ORG_BOOK_APP_URL)
+      console.log('asdasdasd')
 
-      window.location.replace(
-        THE_ORG_BOOK_APP_URL + '/en/recipe/start_a_restaurant?record=' +
-        response.result.id
-      )
+
+      if (response.result == null) {
+        
+                // $(".message").append(" <b>Your request is being processed by one of our representative</b>.");
+
+                alert('Your request is being processed by one of our representative')
+                
+                var delay=2000;
```

Comment:
Is a timeout necessary, or can we have the user click a button to close the window and return to the previous screen.